Core/Core.php

```diff
@@ -26,7 +26,7 @@ function func_num_args () {}
  * The argument offset. Function arguments are counted starting from
  * zero.
  * </p>
- * @return mixed the specified argument, or false on error.
+ * @return mixed|false the specified argument, or false on error.
```

Comment:
This is useless, `false` is a subtype of `mixed`.

Reply:
But instead of reading the phpdoc text, you can now see faster the FALSE type in the IDE ... maybe it's still helpful?